arcgis-ios-sdk-samples/Route & Directions/Route around barriers/RouteAroundBarriersViewController.swift

```diff
@@ -67,8 +67,15 @@ class RouteAroundBarriersViewController: UIViewController, AGSGeoViewTouchDelega
         //get default parameters
         self.getDefaultParameters()
         
+//        //hide directions list
+//        self.setRouteDetailsVisibility(visible: false)
+    }
```

Comment:
The code is commented. Should it be removed?

Reply:
Good catch! Yes, I'll remove it.